### DISCUSSION TITLE
Results in languages other than English are almost impossible?

### DISCUSSION THREAD

**Discussion by Lebski** [serp-ai/bark-with-voice-clone#66] [Q&A]
Hey! 

I finally found a workflow that works quite well for me, but only for English. 
When I try to clone a German voice for example, the results are rather creepy or not in German at all. 

Has anyone else had this experience? Or maybe can even offer a solution? 

Best regards 
Felix

**Comment by dancodedev** [comment]
the same in Spanish

**Comment by devinschumacher** [comment]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik @Lebski  @dancodedev
